README.md

```diff
@@ -466,6 +466,60 @@ Simbody is installed into the `usr/` directory.  The directory
     * `examples/src` source code for the examples.
     * `examples/bin` symbolic link to executable examples.
 
+Windows using MinGW
+-------------------
+
+Warning: The [MinGW](http://sourceforge.net/projects/mingw-w64/)
+generation and build is experimental!
+
+This build is still experimental, because of the precompiled libraries
+Simbody depends on (Blas and Lapack).
+To ease the compilation Windows, Simbody provides precompiled libraries for Blas and Lapack.
```

Comment:
@chrisdembia : for all these typos, should I

- create a new commit for each typo
- create a new commit for all typos
- create and squash all typos in a previous commit